test/base_test.go

```diff
@@ -0,0 +1,125 @@
+package test
+
+import (
+	"context"
+	"github.com/segmentio/kafka-go"
```

Comment:
we should use just the Sarama library and not adding one more dependency for another Kafka client.